src/Dependencies/CodeAnalysis.Debugging/Microsoft.CodeAnalysis.Debugging.Package.csproj

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
+    <PackageId>Microsoft.CodeAnalysis.Debugging</PackageId>
+    <RoslynProjectType>SourcePackage</RoslynProjectType>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Collections" Version="$(SystemCollectionsVersion)" />
+    <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
+    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
+    <PackageReference Include="System.ValueTuple" Version="$(SystemValueTupleVersion)" />
+  </ItemGroup>
+  <Import Project="..\PooledObjects\Microsoft.CodeAnalysis.PooledObjects.projitems" Label="Shared" />
+  <Import Project="$(RepoRoot)build\Targets\SourcePackage.targets"/>
```

Comment:
Can't we auto-import this file in Imports.targets based on `'$(RoslynProjectType)' == 'SourcePackage'"?

Reply:
We can. I guess that would be consistent with how we use RoslynProjectType today.

Reply:
Although I don't quite like the approach of setting a "project type" property and changing bunch of stuff based on that. It's kind of magical. I prefer the opposite - customize projects by importing specific targets. It's easier to track down when you don't know much about the repo as you can just follow the targets.

Reply:
I don't have a strong preference. But today we take the central approach. Prefer to be consistent. If we want to move away we shoudl also be doing it for VSSDK.targets for exampel.

Reply:
I agree with consistency. I'll make the change to use the central approach for now.

Reply:
So, it gets complicated. The target needs to be invoked before common targets. The project is cross-targeting, so we would need to set ```CustomBeforeMicrosoftCommonCrossTargetingTargets``` and import it there. Doesn't feel like it's worth it.

Reply:
I'm fine with this change. Thanks for looking into that.